src/platform/windows/input.cpp

```diff
@@ -200,6 +200,8 @@ namespace platf {
       if (!VIGEM_SUCCESS(status)) {
         BOOST_LOG(warning) << "Couldn't setup connection to ViGEm for gamepad support ["sv << util::hex(status).to_string_view() << ']';
 
+        // Log a special fatal message for this case to show the error in the web UI
+        BOOST_LOG(fatal) << "ViGEmBus is not installed or running. You must install ViGEmBus for gamepad support!"sv;
```

Comment:
Thank you! I think the final vigembus version has been very finicky to install.

Reply:
Yeah, I've seen issues with it too (especially with those unexpected reboots when upgrading ViGEmBus). I'm planning to put up another PR to avoid installing it if it's already present on the system to hopefully avoid these issues.

Reply:
We should probably throw up a warning too, that it's known to unexpectedly reboot during install.

Reply:
The reason it's rebooting is that the driver is already installed and in use, so I think it should never reboot if we don't upgrade existing versions.

Reply:
That makes sense. Can we detect the installed version and put up a warning in the UI if it's not the latest?

We can get latest release info from https://api.github.com/repos/nefarius/ViGEmBus/releases/latest using the `tag_name` key for comparison.